synapse/events/builder.py

```diff
@@ -58,7 +85,29 @@ def create_event_id(self):
 
         return e_id.to_string()
 
-    def new(self, key_values={}):
+    def new(self, room_version, key_values={}):
```

Comment:
one to fix another time, but this default for `key_values` looks disastrous, since we modify it and the modifications will get reused next time it is called.

Reply:
Good point! Will fix that in another PR